### ISSUE TITLE
Issue with Agilent Qtof files

### ISSUE THREAD

**Issue by yguitton** [computational-metabolomics/mspurity-galaxy#42]
Hi Tom,

I hope your fine

I have some Agilent QTof Files to process  and I was trying to use msPurity in W4M, but I face an issue in Galaxy
https://workflow4metabolomics.usegalaxy.fr/u/florence/h/mediana-pos-msmsreadpuritymsp

In Rstudio I managed to process one file so may be it's more a Galaxy issue maybe linked to #41 

```
library(msPurity)
pa<-purityA("LIPO1.mzML",offsets = c(0.65,0.65))
pa@puritydf
```

Any clue?

Thanks for your help
Yann

**Comment by yguitton** [comment]
Hi Tom,

It seems that the error is linked to the minoffset parameter that is not correctly send to Galaxy

```
tool error
An error occurred with this dataset:
cannot find 'minoffset'
```

All the best
Yann

**Comment by Tomnl** [comment]
Hi @yguitton 

Sorry! This should now be fixed in the latest toolshed version of the tools.

It was actually fixed by @bernt-matthias in #41 but I didn't get round to pushing to the toolshed.

Let me know if you get any problems.

**Comment by yguitton** [comment]
Hi @Tomnl,

Many thanks for the tool update. I will ask to have it installed on [usegalaxy.fr](url)

Cheers
Yann
